HW06-Annotation/src/main/java/com/testframework/TestFramework.java

```diff
@@ -0,0 +1,98 @@
+package com.testframework;
+
+import com.akproductions.After;
+import com.akproductions.Before;
+import com.akproductions.Test;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class TestFramework {
+    Class clazz;
```

Comment:
```suggestion
private Class clazz;
```

Поля должны быть приватными.
Инкапсуляция.
Чтобы никто снаружи не мог напрямую изменить состояние класса.